### PR TITLE
watch protocol case fix

### PR DESCRIPTION
**Pull request by ninioe**
fix the Watch Protocol Heart Rate & Blood Pressure Report

### CODE REVIEW

**Comment by macano13** [comment]
Which model of watch do you use ?

File: src/main/java/org/traccar/protocol/WatchProtocolDecoder.java [hunk 1]
```diff
@@ -279,7 +279,7 @@ protected Object decode(
                 String[] values = buf.toString(StandardCharsets.US_ASCII).split(",");
                 int valueIndex = 0;
 
-                if (type.equals("bphrt")) {
+                if (type.toLowerCase().equalsIgnoreCase("BPHRT")) {
```
[the remark this answers is not in the document]
Comment (ninioe) [reply src/main/java/org/traccar/protocol/WatchProtocolDecoder.java]: sorry for this mistake, I must have missed it.
In my watch protocol I also have "BLOOD" key word, can I add it too in the commit?

[the remark this answers is not in the document]
Comment (ninioe) [reply src/main/java/org/traccar/protocol/WatchProtocolDecoder.java]: here is the protocol I got from the supplier
[Y7 Server protocol.en.pdf](https://github.com/traccar/traccar/files/3613426/Y7.Server.protocol.en.pdf)

[the remark this answers is not in the document]
Comment (tananaev) [reply src/main/java/org/traccar/protocol/WatchProtocolDecoder.java]: Sure.

**Comment by ninioe** [comment]
Why does it show failed?

**Comment by tananaev** [comment]
Well, check the details.

**Comment by ninioe** [comment]
I didn’t understand the details.
I’m not sure what to do?

**Comment by tananaev** [comment]
There is "Details" button.

**Comment by ninioe** [comment]
Yes I know, but I didn’t understand what is wrong?

**Comment by tananaev** [comment]
```
> Task :checkstyleMain
[ant:checkstyle] [ERROR] /home/travis/build/traccar/traccar/src/main/java/org/traccar/protocol/WatchProtocolDecoder.java:270: Line is longer than 120 characters (found 154). [LineLength]
```

**Comment by ninioe** [comment]
Hi Anton,
Is there anything else you want me to fix?

**Comment by tananaev** [comment]
Please add a test case for `BLOOD` message type. It has to be a real message you get from your device.

**Comment by ninioe** [comment]
Hi Amton,
Can you add this fix to the next release?

Thanks
Erez

**Pull request merged**